packages/create-svelte/package.json

```diff
@@ -19,6 +19,7 @@
 		"@types/gitignore-parser": "^0.0.0",
 		"@types/prettier": "^2.4.2",
 		"@types/prompts": "^2.0.14",
+		"eslint": "^8.11.0",
```

Comment:
why?

Reply:
This package needs eslint to run `npm run lint`